docs/api.md

```diff
@@ -51,7 +48,40 @@ All the original static methods of the component are hoisted.
 
 ###### `getWrappedInstance(): ReactComponent`
 
-Returns the wrapped component instance. Only available if you pass `{ withRef: true }` as part of the `connect()`’s second `options` argument.
+Returns the wrapped component instance. Only available if you set `connect.defaults({ withRef: true })`.
+
+### `connect.defaults([newDefaults])`
+
+Returns a new `connect` which will have its defaults amended according to `newDefaults`.
+
+Calls to `.defaults()` can be chained, each latter call overriding defaults set in previous ones:
+
+```js
+const withRef = connect.defaults({ withRef: true })
+const withRefAndWithPost = withRef.defaults({ method: 'POST' })
+const onlyWithPost = withRefAndWithPost.defaults({ withRef: false })
+```
+
+#### Arguments
+
+* `[newDefaults = {}]` *(Object)*: An object with any of the following keys:
+     - `buildRequest(mapping): Request` *(Function)*: Takes a mapping and returns a `Request`. If setting this, make sure it interoperates with either the default `fetch` and `Request` or the `fetch` and `Request` you've provided. Defaults to the internal implementation, see the source for details.
+     - `fetch(url|Request, options): Promise<Request>` *(Function)*: The function to use when performing requests. It should conform to the [`window.fetch` API](https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch). If setting this, make sure it interoperates with either the default `Request` or the `Request` you've provided. Defaults to the global `fetch`, if available, checking on `window`, `global`, and `self` in that order.
```

Comment:
Um, not sure which instance of `fetch` you're talking about here.

If the link to MDN: Not sure. I wanted to keep the distinction with the method name and the React Refetch terminology.

If the "...global `fetch`, if available...", that's because it can either be `window.fetch`, `global.fetch`, or `self.fetch`. If that's unclear I can try to reword.